src/cls.ts

```diff
@@ -137,23 +139,25 @@ export class TraceCLS implements CLS<RootContext> {
   }
 
   enable(): void {
-    if (!this.enabled) {
+    // if this.CLSClass = DefaultCLS, the user specifically asked not to use
```

Comment:
So `DefaultCLS` is a CLS that does nothing, am I correct? The name seems confusing then. What about `NullCLS`?